docs/src/main/paradox/routing-dsl/testkit.md

```diff
@@ -9,7 +9,7 @@ route logic easy and convenient. This "route test DSL" is made available with th
 To use Akka HTTP TestKit, add the module to your project:
 
 @@dependency [sbt,Gradle,Maven] {
-  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="2.5.19"
+  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="2.5.23"
```

Comment:
```suggestion
  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="$akka.version$"
```

Reply:
Hmm, the version we recommend in the docs might still be higher than the version we really require though... perhaps we should introduce yet another variable for this?

Reply:
Wouldn't we always recommend the latest? Right now, we seem to use `akka.version` throughout, so maybe let's keep it consistent for now?

I guess we can think about always recommending the latest version of akka?

Reply:
Yeah, not necessary for this PR indeed ;)